distributed/tests/test_scheduler.py

```diff
@@ -1845,6 +1845,37 @@ async def test_get_task_duration(c, s, a, b):
         assert len(s.unknown_durations["slowinc"]) == 1
 
 
+@gen_cluster(client=True)
+async def test_default_task_duration_splits(c, s, a, b):
+    """This test ensures that the default task durations for shuffle split tasks are, by default, aligned with the task names of dask.dask"""
+
+    pd = pytest.importorskip("pandas")
+    dd = pytest.importorskip("dask.dataframe")
+
+    # We don't care about the actual computation here but we'll schedule one anyhow to verify that we're looking for the correct key
+    npart = 10
+    df = dd.from_pandas(pd.DataFrame({"A": range(100), "B": 1}), npartitions=npart)
+    graph = df.shuffle(
+        "A",
+        shuffle="tasks",
+        # If we don't have enough partitions, we'll fall back to a simple shuffle
+        max_branch=npart - 1,
+    ).sum()
+    c.compute(graph)
+
+    while not s.tasks:
+        await asyncio.sleep(0.005)
+
+    prefixes = set(s.task_prefixes.keys())
+    split_prefix = [pre for pre in prefixes if "split" in pre]
```

Comment:
Nitpicks: I don't think we need the extra `set` conversion because `dict` keys should already be unique ([and `set`-like](https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects)). Additionally, since the `prefix` variable is only used in one place, let's inline it there 

```suggestion
    split_prefix = [pre for pre in s.task_prefixes.keys() if "split" in pre]
```

Reply:
It is. I tend to use more variables than necessary since imo this makes it easier to debug. In particular for tests, the performance impact of a set conversion here is negligible. In this case I don't mind the inlining, though